src/term/cell.rs

```diff
@@ -89,12 +93,8 @@ impl Cell {
     /// Get foreground and background colors adjusted for INVERSE flag
     ///
     /// First color is the foreground, second color is the background.
-    pub fn colors(&self, force_invert: bool) -> (&Color, &Color) {
-        if self.flags.contains(INVERSE) || force_invert {
-            (&self.bg, &self.fg)
-        } else {
-            (&self.fg, &self.bg)
-        }
+    pub fn colors(&self) -> (&Color, &Color) {
```

Comment:
Seems like this method isn't doing much anymore; maybe it should just disappear.